### ISSUE TITLE
Fig shows up on the wrong screen

### ISSUE THREAD

**Issue by Lwrless** [withfig/fig#18]
**Summary:**
Fig shows up on the main screen when there's a split-paned fullscreen iTerm2 on a secondary screen. 
Split-pane was achieved with "Split Pane Vertically" and "Split Pane Horizontally" in the right-click menu of iTerm2. And the layout is like this:
```
+--------+--------+
|        |        |
|        +--------|
|--------+        |
|        +--------|
|--------+        |
|        +--------|
|        |        |
+--------+--------+
```

**Steps to Reproduce:**
1. Open iTerm2, make it fullscreen and create split panes.
2. Type a command, Fig shows up on the wrong screen.

**Expected Results:**
Fig shows up right where the command is typed in.

**Actual Results:**
Fig shows up on the edge of my main screen.

**System information:**
OS: macOS Catalina 10.15.7
Terminal: iTerm2
Version of Fig: 1.0.32 (B80)

![Screen Shot 2021-01-25 at 8 58 50 AM](https://user-images.githubusercontent.com/2044349/105650211-fecc6400-5eed-11eb-9437-3047360c91d2.png)

There's a similar problem reported in https://github.com/withfig/fig/issues/10#issuecomment-705124315.

**Comment by mattschrage** [comment]
Thank you for reporting this. I don't have an external monitor right now, so this is somewhat hard for me to debug. I'm planning on getting one shortly and will investigate.

**Comment by mattschrage** [comment]
Took much longer than expected to get that monitor! But this bug is fixed in v1.0.40!
